From 13f33502aa8bdde042f771e1024e1f84dda58f8d Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Sat, 6 Dec 2025 19:48:13 +0800
Subject: [PATCH 082/133] test: remove test requiring 0.16 @Type changes
X-Developer-Signature: v=1; a=openpgp-sha256; l=816; i=xtex@astrafall.org;
 h=from:subject; bh=fIgVFoTCe2XTzjWNupWVK6QoNPOhDcBt88ch4RKw2zk=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78thjW59ExAptNmJLqvml1NGwvjC+vfOwZqK/ApdY9
 rGtrtwdE1kYxLgYLMUUWYoMG7xZddL5RZeVy8LMYWUCGSIt0sDAwMDAwsCXm5hXaqRjpGeqbahn
 aKhjpGPEwMUpAFN94zkjw9JDW3gU+Y4Vyv9k7BL1ky0ts8rcknnywGH3gFrpV7lWlgz/zJZzOSe
 LZdgGX2U/7Ljl6c4rbGsbWkxK+ZgXnwid8qiWHQA=
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

Change-Id: I27f0f5bc96cc8a35a205c2969ec70bf7482c0a55
---
 test/behavior/type.zig | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/test/behavior/type.zig b/test/behavior/type.zig
index 9d65e8e4350d..78295d3a7de2 100644
--- a/test/behavior/type.zig
+++ b/test/behavior/type.zig
@@ -812,12 +812,3 @@ test "undefined type value" {
     };
     comptime assert(@TypeOf(S.undef_type) == type);
 }
-
-test "reify struct with zero fields through const arrays" {
-    const names: [0][]const u8 = .{};
-    const types: [0]type = .{};
-    const attrs: [0]std.builtin.Type.StructField.Attributes = .{};
-    const S = @Struct(.auto, null, &names, &types, &attrs);
-    comptime assert(@typeInfo(S) == .@"struct");
-    comptime assert(@typeInfo(S).@"struct".fields.len == 0);
-}
-- 
2.52.0

